Makefile

```diff
@@ -1,5 +1,5 @@
-short_ver = 1.1.4
-last_ver = 1.1.3
+short_ver = 1.1.5
+last_ver = 1.1.4
```

Comment:
Do we really need a new version here and cannot simply delete and recreate the tag (is there actually a tag?)